ob1k-concurrent/src/main/java/com/outbrain/ob1k/concurrent/ComposableFutures.java

```diff
@@ -164,6 +164,18 @@ private ComposableFutures() {
     return result;
   }
 
+  /**
+   * Continues a future with a handler that will be called only if the original future resulted with success
```

Comment:
description is unclear.
describe here that the purpose of this method is to flatten a nested future. there's no handler.

for example, Scala's description:

  /** Creates a new future with one level of nesting flattened, this method is equivalent
   *  to `flatMap(identity)`.

Reply:
Will change that as well, Thanks for the feedback!